utils/src/ast-grep/remove-lines.ts

```diff
@@ -0,0 +1,20 @@
+import type { Range } from "@ast-grep/napi";
+
+export function removeLines(sourceCode: string, ranges: Range[]) {
+	const lines = sourceCode.split("\n");
+
+	let removeCounter = 0;
+	for (const range of ranges) {
+		// @ts-ignore - @ast-grep/napi is returning start.row, but the type says the field need to be
```

Comment:
@mohebifar I know you worked on codemod and ast-grep any input here ?

Reply:
https://github.com/codemod-com/codemod/pull/1655

I opened a PR on their repo about it

Reply:
I mean adding the pr link the codemod one in the code comment aside of `@ts-ignore`

Reply:
I got it know, I will do it so.